package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "@wormhole-foundation/relayer-engine",
-  "version": "0.1.0",
+  "version": "0.1.3",
   "description": "Relayer Engine",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
     "redis": "docker run --rm -p 6379:6379 --name redis-docker -d redis",
     "test-redis": "docker run --rm -p 6301:6379 --name relayer-engine-test -d redis; npm run test; docker kill relayer-engine-test",
-    "test": "jest --silent=false",
+    "test": "npx jest ./test --silent=false",
```

Comment:
should we move the test inside middleware/missedVaas as well?

Reply:
Before finishing the PR I'll remove the "legacy implementation" because it has at least two bugs. Thanks for the headsup though!